introduction.md

```diff
@@ -1,2 +1,10 @@
 Hello peeps! Your master is in da house!
 The best code school path is iOS.
+
+Deadly Skills:
+* iOS
+* git
+* software engineering
+* problem solving
+* collaboration
+* communication
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?